mkinitrd

```diff
@@ -88,9 +88,12 @@ main() {
 	fi
 
 	install -m755 "$TOPDIR"/init "$TMPDIR"/init
+	
+	[ "" == "$CPIO_TYPE" ] && CPIO_NAME="zstd"
+	[ "" == "$CPIO_PARAM" ] && CPIO_PARAM="-12"
 
 	pushd "$TMPDIR"
-		find . | gcpio -H newc -o | zstd -12 > "$TOPDIR"/seiba.zst
+		find . | gcpio -H newc -o | ${CPIO_TYPE} ${CPIO_PARAM} > "$TOPDIR"/seiba.${CPIO_TYPE}
```

Comment:
Now users can change (shitty) zstd format with ${CPIO_TYPE}

Reply:
Ataraxia uses zstd by default

Reply:
If users not change CPIO_TYPE value
uses zstd

Reply:
and?

Reply:
if you need uncompressed initrd
use CPIO_COMPRESS=cat

Reply:
remove this stuff from commit

Reply:
Removed. https://github.com/ataraxialinux/mkinitrd/pull/1/commits/128354be3ba11ab09b00b1eb3fbd50440ad47979